### ISSUE TITLE
can taso convert old weights to to new weights?

### ISSUE THREAD

**Issue by wangxihuiok** [jiazhihao/TASO#46]
for example, graph optimization can fuse the conv, batchnorm layer to a new conv layer, and calculate the new fused conv weight parameters, can taso calculate the new weight in the transformation after fusing conv and batchnorm?

**Comment by jiazhihao** [comment]
Yes, TASO preprocesses the new weights for the convolution when it is fused with a batchnorm. You can see the preprocessing steps when using TASO to optimize a ConvNet with batchnorm (e.g., ResNet).

Thanks for your interests in TASO.
